apps/verticalhome/js/stores/item.js

```diff
@@ -194,9 +195,13 @@
 
           objectStore.createIndex('index', 'index', { unique: true });
           isEmpty = true;
+
           var objectSV = db.createObjectStore(DB_SV_APP_STORE_NAME,
             { keyPath: 'manifestURL' });
           objectSV.createIndex('indexSV', 'indexSV', { unique: true });
+
+          var pinAppStore = db.createObjectStore(DB_PIN_APP_STORE, { keyPath: 'index'});
+          pinAppStore.createIndex('index', 'index', { unique: true });
```

Comment:
It seems like a bad idea to create an index with the same name as the key that is being indexed.  I can't tell if this index is ever actually used. (Maybe loadTable uses it?)